docs/release/support_matrix_v2_0.md

```diff
@@ -1,73 +1,273 @@
-CentOS Stream and openSUSE Tumbleweed are not supported.
-Other Linux distributions may be supported in the future.
+DAOS Version 2.0 supports the latest RHEL 8 release (currently RHEL 8.5).
+Customers with Extended Update Support (EUS) for RHEL 8 will also be supported
+with the latest RHEL 8 EUS release (currently RHEL 8.4) on the DAOS servers.
+RHEL 8.3 was not an EUS release, and is no longer supported by Red Hat or DAOS.
 
-Please find below the recommended Linux distributions
-that have been tested with the different DAOS release tags.
+Links to RHEL 8 Release Notes:
+* [RHEL 8.3](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.3_release_notes/index)
+* [RHEL 8.4](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.4_release_notes/index)
+* [RHEL 8.5](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.5_release_notes/index)
 
-| **Tag** | CentOS Linux | Rocky/Alma | openSUSE Leap | Ubuntu     |
-|---------|--------------|------------|---------------|------------|
-| v1.0.1  | 7.7          |            | No            | No         |
-| v1.2    | 7.9          |            | No            | No         |
-| v2.0    | 7.9; 8.3/8.4 | No         | 15.2          | No         |
-| v2.2    | 7.9          | evaluating | 15.x          | evaluating |
+Refer to the [RHEL Life Cycle](https://access.redhat.com/support/policy/updates/errata/)
+description on the Red Hat support website for information on RHEL support phases.
+
+### Rocky Linux
+
+DAOS Version 2.0 does not yet support Rocky Linux,
+but Rocky Linux support is targeted for a future DAOS release.
+
+Links to Rocky Linux Release Notes:
+* [Rocky Linux 8.4](https://docs.rockylinux.org/release_notes/8.4/)
+* [Rocky Linux 8.5](https://docs.rockylinux.org/release_notes/8.5/)
+
+### openSUSE Leap 15
+
+DAOS Version 2.0 is supported on openSUSE Leap 15.3.
+
+Links to openSUSE Leap 15 Release Notes:
+* [openSUSE Leap 15.3](https://doc.opensuse.org/release-notes/x86_64/openSUSE/Leap/15.3/)
+
+### SUSE Linux Enterprise Server 15
+
+DAOS Version 2.0 is supported on SLES 15 SP3.
+
+Links to SLES 15 Release Notes:
+* [SLES 15 SP3](https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP3/)
+
+### Ubuntu 20
+
+DAOS Version 2.0 does not support Ubuntu.
+
+### Unsupported Linux Distributions
+
+DAOS does not support
+openSUSE Tumbleweed,
+Fedora,
+CentOS Stream,
+Alma Linux, or
+Oracle Linux.
+
+
+## Operating Systems supported for DAOS Clients
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+In DAOS Version 2.0, the supported Linux distributions and versions for DAOS clients
+are identical to those for DAOS servers. Please refer to the previous section for details.
+
+In future DAOS releases, DAOS client support may be added for additional
+Linux distributions and/or versions.
 
-Information for future releases is indicative only and may change.
```

Comment:
Do we want to leave this in just in case plans with Rocky change? I mean, just in case something got in our way (SDL, etc)?

Reply:
ok, adding the disclaimer back (to the top paragraph)